TwitchLib.Client/Models/OutboundChatMessage.cs

```diff
@@ -8,7 +8,7 @@ public class OutboundChatMessage
 
         public override string ToString()
         {
-            return $":{Username}!{Username}@{Username}.tmi.twitch.tv PRIVMSG #{Channel} :{Message}";
+            return $":{Username.ToLower()}!{Username.ToLower()}@{Username.ToLower()}.tmi.twitch.tv PRIVMSG #{Channel.ToLower()} :{Message}";
```

Comment:
instead of calling ToLower multiple times maybe introduce a intermediate variable like var username = Username.ToLower()